tests/visco_plastic_differentplasticitydifferentphases.prm

```diff
@@ -0,0 +1,105 @@
+# Copy of visco_plastic.prm, but include a phase transition for the EoS
```

Comment:
Can you add one other sentence that says what this test specifically tests? I.e. different plasticity parameters for different phases.